README.md

```diff
@@ -1,5 +1,8 @@
 # KONG Website
 
+[![Build Status](https://api.travis-ci.org/Kong/docs.konghq.com.svg?branch=master)](https://travis-ci.org/Kong/docs.konghq.com/)
```

Comment:
This is showing a build error - but... there is no error?

Reply:
It is not showing any error, the build is passing successfully!